overrides/kubejs/server_scripts/src/ingot_heatcast.js

```diff
@@ -0,0 +1,363 @@
+let gtceuIngots1 = [
+    "aluminium",
```

Comment:
this static list is very bad, you should iterate through all gt materials and check if it has an IngotProperty

Reply:
> this static list is very bad, you should iterate through all gt materials and check if it has an IngotProperty

:saluting_face:

Reply:
> this static list is very bad, you should iterate through all gt materials and check if it has an IngotProperty

How would I do this any more efficiently when all the Materials in the mod are declared individually with no organizational structure, or means of access?